src/zeroD/ReactorNet.cpp

```diff
@@ -148,6 +191,28 @@ double ReactorNet::step()
     return m_time;
 }
 
+void ReactorNet::estimate(double time, int k, double* yest)
+{
+    double* cvode_dky;
+    double deltat = time - m_time;
+    double factor = 1.;
+
+    // initialize
+    cvode_dky = m_integ->solution();
+    for (size_t j = 0; j < m_nv; j++) {
+        yest[j] = cvode_dky[j];
+    }
+
+    // taylor expansion
+    for (int n = 1; n <= k; n++) {
+        factor *= deltat / ((double)n);
```

Comment:
I don't think the explicit cast is necessary - the result of dividing a `double` by an `int` should automatically be a `double`.

Reply:
Old habits. I may misremember compilers complaining eons ago. Will check on this.